### ISSUE TITLE
null pointer exception when media-type attribute not present on manifest item

### ISSUE THREAD

**Issue by mattgarrish** [IDPF/epubcheck#575]
As the subject says, got this:

PS C:\repos\epub3-samples\30> java -jar C:\epubcheck\epubcheck.jar -mode exp .\WCAG\
Validating using EPUB version 3.0 rules.
ERROR(RSC-005): .\WCAG.epub/EPUB/package.opf(69,52): Error while parsing file 'element "item" missing required attribute "media-type"'.
java.lang.NullPointerException
        at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:191)
        at com.adobe.epubcheck.opf.OPFItem$Builder.<init>(OPFItem.java:304)
        at com.adobe.epubcheck.opf.OPFHandler.startElement(OPFHandler.java:328)
        at com.adobe.epubcheck.opf.OPFHandler30.startElement(OPFHandler30.java:124)
        at com.adobe.epubcheck.xml.XMLParser.startElement(XMLParser.java:542)
        at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.startElement(Unknown Source)
        at com.sun.org.apache.xerces.internal.parsers.AbstractXMLDocumentParser.emptyElement(Unknown Source)
        at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.scanStartElement(Unknown Source)
        at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(Unknown Source)
        at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(Unknown Source)
        at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(Unknown Source)
        at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(Unknown Source)
        at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(Unknown Source)
        at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(Unknown Source)
        at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(Unknown Source)
        at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(Unknown Source)
        at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(Unknown Source)
        at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl.parse(Unknown Source)
        at com.adobe.epubcheck.xml.XMLParser.process(XMLParser.java:210)
        at com.adobe.epubcheck.opf.OPFChecker.validate(OPFChecker.java:218)
        at com.adobe.epubcheck.opf.OPFChecker30.validate(OPFChecker30.java:102)
        at com.adobe.epubcheck.opf.OPFChecker.runChecks(OPFChecker.java:132)
        at com.adobe.epubcheck.opf.OPFChecker30.runChecks(OPFChecker30.java:87)
        at com.adobe.epubcheck.ocf.OCFChecker.runChecks(OCFChecker.java:300)
        at com.adobe.epubcheck.api.EpubCheck.doValidate(EpubCheck.java:215)
        at com.adobe.epubcheck.tool.EpubChecker.processFile(EpubChecker.java:456)
        at com.adobe.epubcheck.tool.EpubChecker.run(EpubChecker.java:254)
        at com.adobe.epubcheck.tool.Checker.main(Checker.java:31)
epubcheck completed
